src/StateService/StatePlugin.P2P.cs

```diff
@@ -0,0 +1,30 @@
+using Akka.Actor;
+using Neo.Network.P2P.Payloads;
+using Neo.Plugins.StateService.Storage;
+using System;
+using Neo.IO;
+
+namespace Neo.Plugins.StateService
+{
+    public partial class StatePlugin : Plugin, IP2PPlugin
+    {
+        internal const string StatePayloadCategory = "StateService";
+
+        void IP2PPlugin.OnVerifiedInventory(IInventory inventory)
```

Comment:
We have removed `OnVerifiedInventory` in neo-project/neo#2216

Reply:
Done.